### PR TITLE
Simplified openapi hosting

### PR DESCRIPTION
**Pull request by sakibstark11**
Fixes #
Used when the changes in the pull request directly address the issue.

Resolves #
Used when the changes in the pull request indirectly resolve another issue.

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [x] Improvement
- [ ] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
- OpenApi changes
  - removes bruno as it is deemed unnecessary
  - removes docker-compose to host swagger
  - adds a simple redocly container
  - simplifies the execution of openapi spec json generation by using a directory sha1 only on creation
  - adds S3_HOSTNAME incase newer macos dns creates issues
  - adds stop-dev command
Provide a brief overview of the changes introduced in this pull request.

## Related Issues/PRs (optional)

Link to the relevant issue(s) or provide a clear description of the problem or feature being addressed.

## How to test

- [ ] Needs regression testing (add QA needed label)

## Checklist

- [ ] I have tested these changes locally and they function as expected.
- [ ] I have added or updated necessary documentation to reflect the changes.
- [ ] I have followed the project's coding style and guidelines.
- [ ] I have added appropriate test coverage to ensure the changes are functioning correctly.
- [ ] I have checked for any potential conflicts with other branches or pull requests.
- [ ] I have updated necessary OpenAPI specs

## Reviewer Request (optional)

If you have any specific requests or suggestions for reviewers, mention them here.

## Screenshots (optional)

If applicable, provide any additional screenshots or GIFs that showcase the changes made in this pull request.

## Demo Link (optional)

If applicable, provide a link to a live demo or deployment where the changes can be reviewed.

## Known Issues (optional)

If there are any known issues or limitations with the changes made in this pull request, describe them here.

## Additional Notes (optional)

Add any additional notes or comments you think are relevant to this pull request.

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

### CODE REVIEW

File: openapi/bruno/bruno.json
Comment (melon-ruet) [inline openapi/bruno/bruno.json:1]: The purpose of bruno is to play with API's in local environment as like Postman. Can we do it using redocly?

**Review by KhirulIslam — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -149,21 +150,22 @@ build-runner-image:
...
 start-dev: build-runner-image localstack-start run-command-install-node-packages run-dev
 
 run-dev: run-command-build-node-all run-command-package-all run-command-tf-init \
          run-command-tf-plan-apply run-command-tf-apply replace-dev-url
 
+stop-dev:
+	docker rm -f $(DOCKER_REDOCLY_CONTAINER_NAME) $(DOCKER_DEV_CONTAINER_NAME) $(DOCKER_LOCALSTACK_CONTAINER_NAME)
 
-# Swagger UI
-swagger-ui:
```
Comment (melon-ruet) [inline Makefile:164]: This is unfinished task from @mahinuzzaman , he wants to use swagger as api doc and use it like postman.

**Review by melon-ruet — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (mahinuzzaman) [reply Makefile:164]: @sakibstark11 vai and me decided we wont remove it

Comment (mahinuzzaman) [reply Makefile:164]: @melon-ruet vai, this is now a finished task

**Comment by sakibstark11** [comment]
Will get back to it

**Pull request closed without merging**